advocacy_docs/partner_docs/ChemaxonJChemPostgreSQLCartridge/index.mdx

```diff
@@ -1,14 +1,12 @@
 ---
-title: 'Chemaxon JChem PostgreSQL Cartridge Implementation Guide'
+title: 'Implementing Chemaxon JChem PostgreSQL Cartridge'
 indexCards: simple
 directoryDefaults:
     iconName: handshake
 ---
 
-<p align="center">
-  <img  src="Images/PartnerProgram.png">
-</p>
+![EDB Partner Program logo](images/PartnerProgram.png)
 <h1 style="text-align: center;">EDB GlobalConnect Technology Partner Implementation Guide</h1>
 <h3 style="text-align: center;">Chemaxon JChem PostgreSQL Cartridge</h3>
```

Comment:
@drothery-edb I wasn't sure what to do with the titles here. They are in HTML and look very much like book titles, but I wasn't sure whether I should delete them or use part of it in the description.

Reply:
We decided to look at possible improvements to the landing page format separately.